src/pages/About/index.js

```diff
@@ -0,0 +1,111 @@
+import React from "react";
+import {makeStyles, Toolbar, Typography} from "@material-ui/core";
+
+const useStyles = makeStyles((theme) => ({
+    root: {
+        display: "flex"
+    },
+    content: {
+        maxWidth: "42%",
```

Comment:
This is a really specific number, just checking it looks okay across screen sizes

Reply:
yeah it is! let me know what you reckon. keen that it's not too much of a journey for the eyes from right to left when it's a new line on very blocky text like this.